composer.json

```diff
@@ -10,7 +10,7 @@
     ],
     "require": {
         "php": "^5.6|^7.0",
-        "symfony/symfony": "^2.8|^3.0",
+        "symfony/symfony": "^2.8|^3.0|^4.0",
```

Comment:
Whowa, wait. Are we really going to support 3 major versions at the same time?!

Reply:
this is the testing component only. as long as it works lets allow all of those here. in our bundles, lets drop 2.8 support as needed to avoid too many BC hacks. but if its no hassle, i am happy to support 2.8 LTS.

Reply:
sorry, for that quick hidden change, i will remove it. Was just to test a travis matrix in seo-bundle: https://github.com/symfony-cmf/seo-bundle/pull/336

Reply:
yeah, lets add it later once 4.0 is released as stable.